apps/client/src/utils/passPropsToChildren.ts

```diff
@@ -14,6 +14,7 @@ const passPropsToChildren = (
   const children = Children.map(childrenProp, (child, index) =>
     React.cloneElement(
       child as React.ReactElement<any>,
+      /* istanbul ignore next */
```

Comment:
Why is this ignored?

Reply:
Because I had the feeling we don't need to test this and it's hard to write a test for this. Can you make a simple test for this?

Reply:
I'll add it to my refactors.